src/game_battler.cpp

```diff
@@ -489,6 +489,27 @@ static int AdjustParam(int base, int mod, int maxval, Span<const int16_t> states
 	return value;
 }
 
+int Game_Battler::CalcValueAfterStates(int value, bool lcf::rpg::State::*adj) const {
```

Comment:
Does this logic need to be fully copy pasted like this? It's already existing above in `AdjustParams`.

Can you make this function just call that one?

Do be aware of the min/max clamping behavior